lib/routes/devices-and-sessions.js

```diff
@@ -190,8 +192,9 @@ module.exports = (log, db, config, customs, push, devices) => {
         const uid = sessionToken.uid
         const ip = request.app.clientAddress
         const payload = body.payload
+        const endpointAction = body.endpointAction || 'devicesNotify'
 
-        if (! validatePushPayload(payload)) {
+        if (! validatePushPayload(payload) && endpointAction !== 'accountVerify') {
           return reply(error.invalidRequestParameter('invalid payload'))
         }
```

Comment:
This is going to need a lot more validation, e.g. asserting that the payload is empty when `endpointAction == 'accountverify'`, asserting that `endpointAction` has a known good value, etc.  Are there other sorts of notifications that we might want to forbid this from sending for security reasons?